### PR TITLE
[dns64] set DNS64 default value to 0

### PR DESCRIPTION
**Pull request by xuyirio**
This PR sets DNS64 default value to 0 for all platforms. The DNS64 flag currently installs and configures bind9. When it's ON, the DNS-SD server will not work properly.

It also adds `python3` as a Docker dependency since some tests require it. It seems that python3 was silently installed along with bind9.

### CODE REVIEW

File: etc/docker/Dockerfile [hunk 1]
```diff
@@ -81,13 +81,17 @@ ENV OTBR_OT_BACKBONE_CI_DEPS curl lcov wget build-essential
 ENV OTBR_NORELEASE_DEPS \
   cpputest-dev
 
+# Required for testing
+ENV OTBR_TEST_DEPS python3
```
Comment (simonlingoogle) [inline etc/docker/Dockerfile:85]: seems unrelated change?

Comment (xuyirio) [reply etc/docker/Dockerfile:85]: Seems python3 is installed along with bind9. So tests using python3 cmd will fail when DNS64=0.

Comment (simonlingoogle) [reply etc/docker/Dockerfile:85]: Maybe we can add python3 to OTBR_DOCKER_REQS?

Comment (xuyirio) [reply etc/docker/Dockerfile]: Update. Thanks!

**Review by simonlingoogle — APPROVED** [review]
LGTM 👍

**Review by wgtdkp — APPROVED** [review]

**Review by jwhui — APPROVED** [review]
